.pre-commit-config.yaml

```diff
@@ -1,12 +1,18 @@
+ci:
+    autofix_prs: false
+    autoupdate_schedule: weekly
```

Comment:
Can we do quarterly? Ruff's release frequency otherwise generates update PRs a bit too often to my liking.

Reply:
Certainly (I just copied it from geopandas)

Reply:
```suggestion
    autoupdate_schedule: quarterly
```

Reply:
Ah. We should update it there as well. In any case, we shall enable geopandas org on pre-commit-ci, right now this does nothing :)

Reply:
> right now this does nothing :)

Ah, that's why I couldn't relate with your "generates update PRs a bit too often to my liking" ;)